plugins/api/src/atomic_apply_acl_operations.cpp

```diff
@@ -351,13 +351,14 @@ namespace
     }
 
     // TODO This function should probably deal with remote zones.
```

Comment:
this TODO can go... once we have a test to confirm remote zones are handled.

Reply:
@FifthPotato @FifthPotato Let's capture the need for a federation test in an issue. You can include a link to the TODO comment.

You can implement a non-federated test in c++.

Reply:
> @FifthPotato @FifthPotato Let's capture the need for a federation test in an issue. You can include a link to the TODO comment.
> 
> You can implement a non-federated test in c++.

Oops, missed this comment. I'll pivot to getting a non-federated test going in C++, and make an issue for the federation test.

Reply:
Just to be sure, a "non-federated test in C++" in this context would imply a unit test, right?

Reply:
Yes. You need to use the zone administration library to add a remote zone (w/o connection info) and then create a user in that zone using the user administration library. That will give you the tools to write a test for this PR.

Reply:
Seems we are still awaiting an issue for writing a test for this so that we can update the TODO comment with it. Does that sound about right?

Reply:
I believe the tests included in this PR cover the TODO.

I don't think we have an icommand which uses the atomic ACLs API?

@FifthPotato Thoughts?

Reply:
After some discussion, please do the following:
- Remove TODO
- Open an issue stating we need a federation test for rx_atomic_apply_acl_operations